docs/source/extensions/logging.rst

```diff
@@ -75,55 +104,76 @@ Lightning offers automatic log functionalities for logging scalars, or manual lo
 
 Automatic Logging
 =================
-Use the :func:`~~pytorch_lightning.core.lightning.LightningModule.log`
-method to log from anywhere in a :doc:`lightning module <../common/lightning_module>` and :doc:`callbacks <../extensions/callbacks>`
-except functions with `batch_start` in their names.
+Use the :meth:`~pytorch_lightning.core.lightning.LightningModule.log`
+method to log from anywhere in a :doc:`lightning module <../common/lightning_module>` and :doc:`callbacks <../extensions/callbacks>`.
 
 .. code-block:: python
 
     def training_step(self, batch, batch_idx):
         self.log("my_metric", x)
 
 
-    # or a dict
+    # or a dict to get multiple metrics on the same plot of the logger supports it
     def training_step(self, batch, batch_idx):
         self.log("performance", {"acc": acc, "recall": recall})
```

Comment:
Or use:

```py
self.log("performance/acc", acc)
self.log("performance/recall", recall)
```

Reply:
no.. this is different. Your example will create `performance` and will create separate plots for each for acc and recall. The existing example will plot recall and acc on the same plot titled `performance`.